src/AVFoundation/AVCompat.cs

```diff
@@ -217,8 +217,15 @@ public virtual void RespondByRequestingPersistableContentKeyRequest ()
 
 #if TVOS
 	// tvOS removed some types - we need to keep stubs of them for binary compatibility
-	[Obsolete ("Removed in tvOS 10.")]
+#if NET
```

Comment:
This entire file is inside a !NET, so there's no need to change anything in here:

https://github.com/xamarin/xamarin-macios/blob/c61ba7112545276d5ce818f62d236cefef88e847/src/AVFoundation/AVCompat.cs#L3

Reply:
Nice catch

Reply:
Corrected.